xml/admin_operating_services.xml

```diff
@@ -233,4 +233,127 @@ ceph-rbd-mirror@.service
 <screen>&prompt.smaster;salt-run state.orch ceph.restart.ganesha</screen>
   </sect2>
  </sect1>
+ <sect1 xml:id="ceph-cluster-shutdown">
+  <title>Shutdown and Start of the Whole &ceph; Cluster</title>
+
+  <para>
+   There are occasions when you need to stop all &ceph; related services in the
```

Comment:
I think this would be better with a small reword.

The text after "for example" is a sentence fragment; therefore, I suggest removing the full stop before it and replacing with a comma or a semi-colon, or even an en dash.

The phrase about "stopping all the services in the correct order" -- this is a clarification. I would remove it from that sentence. You just want to shut down cleanly and safely, right? So what this section provides is instructions how to do it in the correct order.

I was about to add a suggestion but I see that @jan--f already has done so. I endorse that.